### PR TITLE
chore: export artifacts from each package

### PR DESCRIPTION
**Pull request by b00ste**
# What does this PR introduce?

## 📦 Build

Export artefacts in each package

### PR Checklist

- [ ] Wrote Tests
- [ ] Wrote & Generated Documentation (readme/natspec/dodoc)
- [ ] Ran `npm run lint` && `npm run lint:solidity` (solhint)
- [ ] Ran `npm run format` (prettier)
- [ ] Ran `npm run build`
- [ ] Ran `npm run test`

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-10235 Setup artifacts per package](https://app.clickup.com/t/2645698/DEV-10235)

File: packages/lsp6-contracts/hardhat.config.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ const config: HardhatUserConfig = {
   },
   packager: {
     // What contracts to keep the artifacts and the bindings for.
-    contracts: [],
+    contracts: ['LSP6KeyManager', 'LSP6KeyManagerInit'],
```
Comment (CJ42) [inline packages/lsp6-contracts/hardhat.config.ts:110]: Suggest doing a bit like file below to make it look clean and consistent.

```
contracts: [
    'LSP6KeyManager', // Standard version
    'LSP6KeyManagerInit' // Proxy version
],
```

File: packages/lsp7-contracts/hardhat.config.ts
Comment (CJ42) [inline packages/lsp7-contracts/hardhat.config.ts:1]: Super 👌

File: packages/lsp9-contracts/hardhat.config.ts [hunk 2]
```diff
@@ -107,7 +107,7 @@ const config: HardhatUserConfig = {
   },
   packager: {
     // What contracts to keep the artifacts and the bindings for.
-    contracts: [],
+    contracts: ['LSP9Vault', 'LSP9VaultInit'],
```
Comment (CJ42) [inline packages/lsp9-contracts/hardhat.config.ts:110]: Same as suggested above

**Review by CJ42 — CHANGES_REQUESTED** [review]
Added review comments.

Maybe for each individual LSP package only (not the top level `@lukso/lsp-smart-contracts`, we could add the Typechain types of the contracts being included in the `artifacts`?
Typechain types are big if there are a lot, but for artifacts like the LSP6 Key Manager, it should be smaller no?

Let's try to see the package(s) size

**Review by CJ42 — APPROVED** [review]

**Pull request merged**